var/spack/repos/builtin/packages/py-nbconvert/package.py

```diff
@@ -75,13 +80,69 @@ class PyNbconvert(PythonPackage):
     depends_on("py-testpath", when="@5:6.4", type=("build", "run"))
     depends_on("py-lxml", when="@6.5.1:7.0", type=("build", "run"))
 
+    conflicts("^bleach@5.0.0", when="@7.14:")
```

Comment:
```suggestion
    conflicts("^bleach@5.0.0")
```
This is likely true for all versions, but wasn't noticed until bleach 5.0.0 was released